src/meshdb/settings.py

```diff
@@ -172,9 +188,10 @@
 # https://django-dbbackup.readthedocs.io/en/master/installation.html
 
 DBBACKUP_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+DBBACKUP_FILENAME_TEMPLATE = "{datetime}.{extension}"
 DBBACKUP_STORAGE_OPTIONS = {
-    "bucket_name": os.environ.get("BACKUP_S3_BUCKET_NAME"),
-    "location": os.environ.get("BACKUP_S3_BASE_FOLDER"),
+    "bucket_name": "meshdb-data-backups",
```

Comment:
If these env vars are all the way gone, please remove from helm + CI

Reply:
I could go either way on continuing to reference as env vars

Reply:
I'd prefer to clean them up in a follow-up PR. I'd like to make sure everything works properly, first.

Reply:
https://github.com/nycmeshnet/meshdb/issues/535